themes/dco/templates/footer.ftl

```diff
@@ -29,6 +30,39 @@
             <li role="listitem"><a href="http://www.vivoweb.org/support" target="blank" title="${i18n().menu_support}">${i18n().menu_support}</a></li>
         </ul>
     </nav>
+    -->
+    <table style="background-color: black;">
+	<tr>
+	    <td style="width:15%;padding:1em;">
+<ul style="list-style-type:circle;">
+<li class="boiler"><a class="dco_anchor" href="/vivo/" title="">Home</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net/about/about-dco" title="">About</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net" title="">Community Portal</a></li>
+<li class="boiler"><a class="dco_anchor" href="/vivo/" title="">Data Portal</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net/content/news" title="">News</a></li>
```

Comment:
VIVO is https://info.deepcarbon.net so the https://deepcarbon.net links wouldn't be relative, and is why I open them in a new tab/window.